glance/files/mitaka/glance-api.conf.Debian

```diff
@@ -799,55 +799,101 @@ stores = {{ server.storage.engine }}
 # (dict value)
 #http_proxy_information =
 
+{%- if 'swift' in storage_engines %}
+
 # If True, swiftclient won't check for a valid SSL certificate when
 # authenticating. (boolean value)
+{% if server.storage.swift_store_auth_insecure is defined %}
+swift_store_auth_insecure = {{ server.storage.swift_store_auth_insecure }}
```

Comment:
Will it accept upper-case boolean? Eg. `True`? If not or you are not sure, use jinja `|lower` filter.